gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/desktop/application/internal/ComposeProjectProperties.kt

```diff
@@ -51,4 +52,7 @@ internal object ComposeProperties {
 
     fun checkJdkVendor(providers: ProviderFactory): Provider<Boolean> =
         providers.valueOrNull(CHECK_JDK_VENDOR).toBooleanProvider(true)
+
+    fun generateResourcesCertainly(providers: ProviderFactory): Provider<Boolean> =
```

Comment:
generateResourcesCertainly->alwaysGenerateResourceAccessor
and the like (GENERATE_RESOURCES_CERTAINLY)